### DISCUSSION TITLE
tdcs optimization electrode information

### DISCUSSION THREAD

**Discussion by EFraats** [simnibs/simnibs#431] [8. Simulation workflows]
I have two questions regarding electrode stimulation using tDCS optimization:

Is there a way to...:

1. set a minimum number of electrodes for the tDCS optimization process.
2. define a minimum current for each electrode.
3. Ensure the same electrical current is applied to all electrodes.

**Comment by TorgeW** [comment]
Please have a look here:
https://simnibs.github.io/simnibs/build/html/documentation/opt_struct/tdcsoptimize.html#tdcsoptimize-doc
You can set these:
max_total_current
max_individual_current
max_active_electrodes

Setting the minimum number of active electrodes is not possible, defining a minimum current is not possible.
Setting the same current for all of them is also not possible. 

Do you have any special set up where you need this?
